test/object.cpp

```diff
@@ -236,6 +236,21 @@ BOOST_AUTO_TEST_CASE(univalue_array)
     BOOST_CHECK_EQUAL(arr[9].getValStr(), "1");
     BOOST_CHECK_EQUAL(arr[9].getType(), UniValue::VBOOL);
 
+    // Test that range-based for loops work as expected
+    int i{0};
+    for (auto& element : arr) {
```

Comment:
how is this different from
```suggestion
    for (auto& element : arr.getValues()) {
```

Reply:
Ah. I somehow wasn't aware of that.